tests/opt-testcases/tls13-misc.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/sh
+
+# tls13-misc.sh
+#
+# Copyright The Mbed TLS Contributors
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+requires_gnutls_tls1_3
```

Comment:
I am a bit lost with these new tests. Why are they not in `tls13-kex-modes.sh` ? What do the bring in addition to the ones we have in `tls13-kex-modes.sh` ?

Reply:
I plan to move some test cases from `tls13-kex-modes.sh` to here, `tls13-kex-modes.sh` should be generated by scripts.

Reply:
And I think the name `tsl13-kex-modes` does not match the test cases in the file